src/autorest-core/legacyCli.ts

```diff
@@ -60,7 +59,7 @@ export async function CreateConfiguration(baseFolderUri: string, inputScope: Dat
   const codegenerator = switches["g"] || switches["codegenerator"] || "CSharp";
   const usedCodeGenerator = codegenerator.toLowerCase().replace("azure.", "").replace(".fluent", "");
   if (codegenerator.toLowerCase() === "none") {
-    result["azure-validator"] = true;
+    (<any>result)["azure-validator"] = true;
```

Comment:
no longer part of the data type since the core shouldn't care about this - this right here is the single exception. This code translates legacy CLI to an in-memory config - something that doesn't have to be over-engineered if you ask me.